src/StandardReactiveSocket.cpp

```diff
@@ -213,13 +213,16 @@ void StandardReactiveSocket::createResponder(
   debugCheckCorrectExecutor();
 
   if (streamId != 0) {
+// TODO: Remove this: https://github.com/ReactiveSocket/reactivesocket-cpp/issues/243
+#ifndef DONT_VERIFY_STREAM_IDS
```

Comment:
Is this really safe given the bump to lastPeerStreamId_ could now be a stream that we created?  Or we live for this assuming a one directional use case?